templates/html/analysis_periodic_notification.tmpl

```diff
@@ -0,0 +1,9 @@
+{{ template "header" . }}
+
+<p>The analysis <b>{{.analysisname}}</b> is still running and has been running for <b>{{.runduration}}</b>. This is a regularly scheduled courtesy reminder to ensure you don't use up your quota!</p>
```

Comment:
This line is the only thing in here different from the status change email template.